### PR TITLE
Add: ephemeral workload will shutoff

### PR DESCRIPTION
**Pull request by matfechner**

### CODE REVIEW

File: docs/content/en/compute/instance_types.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ Please note that local storage is ephemeral, which means that data
 stored on these volumes is lost if the instance is destroyed.
 
 Data will also not be migrated or evacuated when maintenance tasks are
-performed.
+performed. When maintenance tasks happend these instances will shutoff.
```
Comment (wwentland) [inline docs/content/en/compute/instance_types.md:133]: Could we make this something like "During maintenance windows, instances may be restarted." or "Instances may be temporarily unavailable and/or rebooted during maintenance windows." ?

**Review by wwentland — COMMENTED** [review]
Thanks, that's indeed important information!

File: docs/content/en/compute/instance_types.md [hunk 2]
```diff
@@ -130,7 +130,8 @@ Please note that local storage is ephemeral, which means that data
 stored on these volumes is lost if the instance is destroyed.
 
 Data will also not be migrated or evacuated when maintenance tasks are
-performed.
+performed. During the maintenance windows happend, these instances may be 
+temporarily unavailable and restarted.
```
Comment (wwentland) [inline docs/content/en/compute/instance_types.md:134]: I suggest to remove `happend` here.

**Comment by matfechner** [comment]
obselete or double

**Pull request closed without merging**